intro-to-git/activity-git-history.md

```diff
@@ -0,0 +1,95 @@
+# Activity: Git History
+
+<!-- Written to be synchronous -->
+<!-- Ideal Format: Small groups -->
+
+## Goal
+
+Our goal is to practice the process of making commits as a group. We will also practice collaborating in Git as a group.
+
+Our goal is to practice this without actually typing and running these commands in the terminal; however, we should practice recognizing what commands we _would_ type.
```

Comment:
I'll reconsider my thoughts since cwboden mentioned this too, but I'm hoping for an in-person group activity will be helpful to get them to talk through process before muscle memory

Reply:
After reading the rest of the activity, and the following practice-git-commands activity, I think this is fine as is.